zerver/lib/bugdown/__init__.py

```diff
@@ -146,6 +146,7 @@ def add_a(root, url, link, title=None, desc=None,
         a.set("data-id", data_id)
     img = markdown.util.etree.SubElement(a, "img")
     img.set("src", url)
+    img.set("alt", title)
```

Comment:
I feel like this is another case where I want us to determine what the correct accessibility thing to do is.  Do we need to set  `title`  and `alt` to be the same thing for both this and emoji?  

@jmbowman, can you help track down a good recommendation for how images should be tagged for accessibility that also want a hover-title behavior?  Since changing the HTML for inline images is a somewhat disruptive change, I want to make sure we do it correctly.